airflow/models/renderedtifields.py

```diff
@@ -0,0 +1,153 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Save Rendered Template Fields """
+import json
+from typing import Optional
+
+from sqlalchemy import JSON, Column, String, and_, exists, tuple_
+from sqlalchemy.orm import Session
+
+from airflow.configuration import conf
+from airflow.models.base import ID_LEN, Base
+from airflow.models.taskinstance import TaskInstance
+from airflow.serialization.serialization_helpers import serialize_template_field
```

Comment:
```suggestion
from airflow.serialization.helpers import serialize_template_field
```

I'm not a fan of duplication/redundancy in package names

Reply:
Updated